extras/operator/jsonnet/grafana.libsonnet

```diff
@@ -0,0 +1,17 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+
+{
+  _config+:: {
+    namespace: 'default',
+  },
+  grafana+:: {
+    dashboardDefinitions:
+      local configMap = k.core.v1.configMap;
+      [
+        local dashboardName = 'grafana-dashboard-' + std.strReplace(name, '.json', '');
+        configMap.new(dashboardName, { [name]: std.manifestJsonEx($._config.grafana.dashboards[name], '    ') }) +
+        configMap.mixin.metadata.withNamespace($._config.namespace)
+        for name in std.objectFields($._config.grafana.dashboards)
```

Comment:
I think `$._config.grafana.dashboards ` does not exit as the $._config refers to the config mentioned in line 4. Also I do not understand the complete intent of the code here. It would be better to have some comments here.